### PR TITLE
13968: Updated logic to get the template name in order to tweets to work

### LINKED ISSUE #13968: Unable to find template "@MauticSocial/list.html.twig"
### Mautic Version

5.0.x series

### Way of installing

I downloaded a release from https://www.mautic.org/mautic-releases

### PHP version

8.0.30

### What browsers are you seeing the problem on?

Chrome

### What happened?

A bug happened!

### How can we reproduce this issue?

Step 1:  I activated and configured the Twitter plugin
I get a 500 error on the link {url}/s/tweets
by clicking on the Channels menu >> Tweets
in the logs, I have the following detail

[2024-07-16T15:01:52.867309+00:00] mautic.CRITICAL: Uncaught PHP Exception Twig\Error\LoaderError: "Unable to find template "@MauticSocial/list.html.twig" (looked into: ... /mautic/plugins/MauticSocialBundle/Resources/views)." at .../mautic/vendor/twig/twig/src/Loader/FilesystemLoader.php line 227 {"exception":"[object] (Twig\\Error\\LoaderError(code: 0): Unable to find template \ "@MauticSocial/list.html.twig\" (looked into: .../mautic/plugins/MauticSocialBundle/Resources/views at .../mautic/vendor/twig/twig/src/Loader/FilesystemLoader.php). :227)"} {"hostname":"...","pid":2207479}

### Relevant log output

_No response_

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

### PR DESCRIPTION
**Pull request by shinde-rahul**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes [#13968](https://github.com/mautic/mautic/issues/13968)  

## Description

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. I activated and configured the Twitter plugin
3. Got tho Tweet channel
4. You will be able to add/update tweets as expected.

### CODE REVIEW

File: .ddev/config.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+name: mautic-5
```
Comment (shinde-rahul) [inline .ddev/config.yaml]:
```suggestion
```

File: .ddev/config.yaml [hunk 2]
```diff
@@ -7,8 +8,8 @@ router_https_port: "443"
 xdebug_enabled: false
 additional_hostnames: []
 additional_fqdns: []
-mariadb_version: "10.3"
-mysql_version: ""
+mariadb_version: ""
+mysql_version: "8.0"
```
Comment (shinde-rahul) [inline .ddev/config.yaml]:
```suggestion
mariadb_version: "10.3"
mysql_version: ""
```

**Review by andersonjeccel — APPROVED** [review]
Works as expected! Thanks for fixing this

Before:

image

After:

image

**Review by abhisekmazumdar — APPROVED** [review]
I was able to create tweets. Also, the code makes sense to me.

**Pull request merged**